db/migrate/20150713143226_add_unique_to_operatingsystems_title.rb

```diff
@@ -0,0 +1,29 @@
+class AddUniqueToOperatingsystemsTitle < ActiveRecord::Migration
+  def up
+    dups = Operatingsystem.all.group_by(&:fullname).values.keep_if { |ary| ary.length > 1}
+    dups.each do |systems|
+      systems.each_with_index do |os, index|
+        os.name = "#{index + 1}-" + os.name
```

Comment:
Should we avoid renaming one of the operating systems at least?  Perhaps the one with the highest hosts_count?